src/index.ts

```diff
@@ -1,7 +1,13 @@
 import {Client} from 'rpc-websockets'
 type Hash = string
 
-export const connect = (url) => new Promise((fulfill, reject) => {
+type hcWebClientConnect = {
+  call: (callStr: string) => (params: any) => Promise<string>,
```

Comment:
```suggestion
  call: (...methodSegments: string) => (params: any) => Promise<string>,
```

The first function application can have any number of args now, they all get joined together with "/"